src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React from  'react';
+
+import styled from 'styled-components';
+import Menu from './Menu';
+import SearchBar from './SearchBar';
+import RightHand from './RightHand';
+
+
+const HeaderBox = styled.div`
+    position: sticky;
+    display: flex;
+    width: 100%;
+    background-color: purple;
+    flex-direction: row;
+`;
+
+const Header = (props) => {
```

Comment:
It is better to destructure the props before using them, so you don't have to be using `props.` whenever you want to use a prop. This could be done as:

```javascript
const Header = ({ callBack }) => {
...
```

for example. Now that you are using props, it could be useful to put in place a set of rules for those props. In this case, this is a function, you could add a rule for that prop so that the component expects exactly that. Check the documentation for these and let me know if you have any questions:

https://reactjs.org/docs/typechecking-with-proptypes.html